app/templates/content/framework-g-cloud.html

```diff
@@ -0,0 +1,120 @@
+        </li>
+        <li>
+          the Crown Hosting Data Centres framework, which provides access to physical datacentre space for legacy systems
+        </li>
+      </ul>
+
+
+
+
+
+
+
+
+
+
+      <h2 id="the-g-cloud-framework">
+        <span class="number">3. </span>The G-Cloud framework
+      </h2>
+      <p>
+        The G-Cloud framework is an agreement between the government and suppliers who provide cloud-based services. G-Cloud services are divided into four lots:
+      </p>
+      <ul>
+        <li>
+          <h4>Infrastructure as a Service</h4>
+          Infrastructure is the hardware that makes software work. It’s the networks, hosting facilities and servers on which platforms and software depend. Infrastructure as a Service (IaaS) is infrastructure you can order and run entirely over the internet, without having to pay for your own hardware.
+        </li>
+        <li>
+          <h4>Platform as a Service</h4>
+          Platform as a Service (PaaS) technologies are software platforms that provide a basis for building other services and applications. With PaaS, you can set up, order, pay for and manage platforms in the cloud.
+        </li>
+        <li>
+          <h4>Software as a Service</h4>
+          Software as a Service (SaaS) is an application or service that can be run over the internet or in the cloud. Examples of SaaS include web-based email services, customer relationship management (CRM) software and analytics tools.
+        </li>
+        <li>
+          <h4>Specialist Cloud Services</h4>
+          Specialist Cloud Services (SCS) support your transition to SaaS, PaaS and IaaS. Examples of SCS include cloud strategy, data transfer between providers or day-to-day support of cloud-based services.
+        </li>
+      </ul>
+
+
+
+
+      <h2 id="which-framework-agreement">
+        <span class="number">4. </span>Which framework agreement?
+      </h2>
+      <p>
+        Cloud services on the Digital Marketplace will be on either <a href="http://ccs.cabinetoffice.gov.uk/contracts/rm1557v">G-Cloud 5</a> or <a href="http://ccs-agreements.cabinetoffice.gov.uk/contracts/rm1557vi">G-Cloud 6</a>. G-Cloud 6 is the latest framework agreement.
+      </p>
+      <p>
+        If you’d like to buy technology services from another framework, please email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a> or <a href="http://ccs-agreements.cabinetoffice.gov.uk/">find an agreement</a> on the Crown Commercial Service (CCS) website.
+      </p>
+
+
+
+
+      <h2 id="submit-services-to-the-G-Cloud-framework">
+        <span class="number">5. </span>Submit services to the G-Cloud framework
+      </h2>
+      <p>
+        Read <a href="/suppliers-guide/">the G-Cloud suppliers’ guide</a> to learn how to submit services to the G-Cloud framework.
```

Comment:
It will redirect, but you're right, better to point to the correct link. Addressed in d4d59ea.